app/data/generators/events.js

```diff
@@ -1,9 +1,17 @@
+const { DateTime } = require('luxon')
+
+function getDate(faker, date = new Date, min = 1, max = 20) {
```

Comment:
Must we pass in faker to this? Can we create a new instance? Perhaps we should not be passing in faker at all.

Reply:
I've updated as suggested